packages/synapse-sdk/src/telemetry/get-sentry.ts

```diff
@@ -0,0 +1,27 @@
+import type * as SentryBrowser from '@sentry/browser'
+import type * as SentryNode from '@sentry/node'
+
+// Dynamically import the correct Sentry package based on environment
+const isBrowser =
+  typeof (globalThis as any).window !== 'undefined' && typeof (globalThis as any).document !== 'undefined'
+
+export type Sentry = typeof SentryNode.default | typeof SentryBrowser.default
+
+export async function getSentry(): Promise<{ Sentry: Sentry; integrations: any[] }> {
+  if (isBrowser) {
+    const SentryBrowser = await import('@sentry/browser')
+    return {
+      Sentry: SentryBrowser,
+      integrations: [
+        SentryBrowser.browserTracingIntegration({
+          ignoreResourceSpans: ['resource.script', 'resource.img', 'resource.css', 'resource.link'],
+        }),
+      ],
+    }
+  }
+  const SentryNode = await import('@sentry/node')
+  return {
+    Sentry: SentryNode,
+    integrations: [SentryNode.httpIntegration()],
```

Comment:
Is this even needed?  It looks to be enabled by default per https://docs.sentry.io/platforms/javascript/guides/connect/configuration/integrations/http/

Although it's confusing that it isn't listed at https://docs.sentry.io/platforms/javascript/configuration/integrations/

Reply:
removed